inference/core/workflows/core_steps/cache/cache_get/v1.py

```diff
@@ -0,0 +1,93 @@
+from typing import Any, List, Literal, Optional, Type, Union
+
+from pydantic import ConfigDict, Field
+
+from inference.core.workflows.core_steps.cache.memory_cache import WorkflowMemoryCache
+from inference.core.workflows.core_steps.common.entities import StepExecutionMode
+from inference.core.workflows.execution_engine.entities.base import (
+    OutputDefinition,
+    WorkflowImageData,
+)
+from inference.core.workflows.execution_engine.entities.types import (
+    IMAGE_KIND,
+    STRING_KIND,
+    WILDCARD_KIND,
+    Selector,
+)
+from inference.core.workflows.prototypes.block import (
+    BlockResult,
+    WorkflowBlock,
+    WorkflowBlockManifest,
+)
+
+LONG_DESCRIPTION = """
+Fetches a previously stored value from a cache entry.
+
+Use the `Cache Set` block to store values in the cache.
+"""
+
+SHORT_DESCRIPTION = "Fetches a previously stored value from a cache entry."
+
+PATTERN_STR = r"(^\$inputs.[A-Za-z_0-9\-]+$)"
```

Comment:
Trying to enforce that they use one of the input images vs the output of another step. If video_metadata exists it will exist on those images. But other images in the Workflow may not have it.

Reply:
Sounds good; rolling with that.